chips/cc26x2/src/prcm.rs

```diff
@@ -0,0 +1,380 @@
+//! Power, Clock, and Reset Management (PRCM)
+//!
+//! For details see pg 451-566 of the cc13x2/cc26x2 Technical Reference Manual.
+//!
+//! PRCM manages different power domains on the boards, specifically:
+//!
+//!     * RF Power domain
+//!     * Serial Power domain
+//!     * Peripheral Power domain
+//!
+//! It also manages the clocks attached to almost every peripheral, which needs to
+//! be enabled before usage.
+//!
+use kernel::common::registers::{ReadOnly, ReadWrite, WriteOnly};
+use kernel::common::StaticRef;
+
+#[repr(C)]
+struct AonWucRegisters {
```

Comment:
I can't find a reference to this anywhere in the datasheet. Can you add a pointer in the comments?

Reply:
Its `wake-up-control`. These are the same registers as PMCtl registers set in AON. It's needed here for clock access when forcing shutdown. Another not so clear portion of the tech ref manual. If the LF clock is not deactivated (wherever you are sourcing it from), the mcu will not power down correctly. 

I can add comments in to explain this and some reference page numbers